dialogues/risawoz/src/knowledgebase/en_zh_mappings.py

```diff
@@ -198,137 +278,246 @@ def __init__(self):
-            '倒车影像': 'parking_assist_system',
-            '定速巡航': 'cruise_control_system',
-            '动力水平': 'power_level',
-            '车系': 'series',
-            '厂商': 'brand',
-            '出发时间': 'departure_time',
-            '票价': 'ticket_price',
-            '舱位档次': 'class_cabin',
-            '时长': 'duration',
-            '到达时间': 'arrival_time',
-            '车次信息': 'train_number',
-            '出发地': 'departure',
-            '准点率': 'punctuality_rate',
-            '航班信息': 'flight_information',
-            '坐席': 'seat_type',
-            '制片国家/地区': 'production_country_or_area',
-            '豆瓣评分': 'Douban_score',
-            '片名': 'title',
-            '主演名单': 'name_list',
-            '具体上映时间': 'release_date',
-            '导演': 'director',
-            '片长': 'film_length',
-            '类型': 'type',
-            '年代': 'decade',
-            '主演': 'star',
-            '价格区间': 'pricerange',
-            '内存容量': 'memory_capacity',
-            '商品名称': 'product_name',
-            '显卡型号': 'GPU_model',
-            '裸机重量': 'weight',
-            '品牌': 'brand',
-            '系统': 'operating_system',
-            'CPU型号': 'CPU_model',
-            '系列': 'series',
-            '特性': 'features',
-            '屏幕尺寸': 'screen_size',
-            '游戏性能': 'game_performance',
-            '分类': 'usage',
-            '产品类别': 'computer_type',
-            '待机时长': 'standby_time',
-            '色系': 'colour',
-            '硬盘容量': 'hard_disk_capacity',
-            'CPU': 'CPU',
-            '显卡类别': 'GPU_category',
-            '首播时间': 'premiere_time',
-            '集数': 'episodes',
-            '单集片长': 'episode_length',
-            '开始日期': 'start_date',
-            '老师': 'teacher',
-            '教师': 'teacher',
-            '上课方式': 'type',
-            '校区': 'campus',
-            '难度': 'level',
-            '时段': 'time',
-            '课时': 'hours',
-            '科目': 'subject',
-            '班号': 'class_number',
-            '结束日期': 'end_date',
-            '年级': 'grade',
-            '下课时间': 'end_time',
-            '教室地点': 'classroom',
-            '每周': 'day',
-            '课次': 'times',
-            '上课时间': 'start_time',
-            '星级': 'star',
-            '地铁是否直达': 'metro_station',
-            '停车场': 'parking',
-            '酒店类型': 'hotel_type',
-            '起飞时间': 'departure_time',
-            '营业时间': 'business_hours',
-            '人均消费': 'per_capita_consumption',
-            # below are google translated
-            "车身尺寸(mm)": "body_size(mm)",
-            "发动机排量(L)": "engine_displacement(L)",
-            "发动机马力(Ps)": "engine_horsepower(Ps)",
-            "综合油耗(L/100km)": "comprehensive_fuel_consumption(L/100km)",
-            "环保标准": "environmental_standards",
-            "驾驶辅助影像": "driving_assistance_video",
-            "巡航系统": "cruise_system",
-            "价格(万元)": "price",
-            "地铁线路": "subway_line",
-            "课程网址": "course_URL",
-            "教师网址": "teacher_URL",
+        self.zh2en_SLOT_MAPS = {
```

Comment:
Thanks for your changes in this file to make it compatible for other languages. 
Since we're retraining the English slot and domain names for other languages, there's no need to change `self.zh2en_DOMAIN_MAPS` and `self.zh2en_SLOT_MAPS`. 
The rest of the changes in the file look good.

Reply:
Do you have canceled these changes already or should I do it in a new commit?

Reply:
I have made this change. I will push it soon.